app/src/main/res/layout/fragment_r_details.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:background="@color/white"
+    android:layout_width="match_parent"
+    android:gravity="center"
+    android:orientation="vertical"
+    android:layout_height="match_parent"
+    tools:context=".rdashboard.rdetailedPosts.RDetailsFragment">
+
+<FrameLayout  android:layout_width="match_parent"
+    android:layout_height="240dp">
+
+    <ImageView
+        android:layout_width="match_parent"
+        android:scaleType="fitXY"
+        android:adjustViewBounds="true"
+        android:gravity="center"
+        android:src="@drawable/pizza"
```

Comment:
Why use a default drawable?

Reply:
To see how the uploaded image looks like, i used default drawable.